client/src/models/results.js

```diff
@@ -311,10 +311,10 @@ class Indicator {
   static create_and_register(def){
     const { id, result_id } = def;
 
-    if(this.lookup(id)){
+    if(this.lookup(id).doc){ // hack to allow indicators as subjects before they're loaded
```

Comment:
As discussed, these hacks won't be needed if the individual indicator route makes its own GraphWL queries.